### ISSUE TITLE
Single column experiments

### ISSUE THREAD

**Issue by willaguiar** [willaguiar/DSW-collaborative-project#34]
Issue to discuss the results of single column convection experiments, and better ways to set them up.

**Background**: We found a [contrasting response of dense water formation](https://github.com/willaguiar/DSW-collaborative-project/issues/32) to thickening the top ocean model level in  the North Atlantic and Southern Ocean. One possible reason for the contrasting response is that in the Southern ocean, convection and SWMT  is salt-driven and in the North Atlantic convection is heat-driven. To separate the the response of surface $\sigma_0$ to thickening the top ocean cell in salt-driven and heat-driven convection settings, we set up some ocean-only, [single column](https://github.com/NOAA-GFDL/MOM6-examples/tree/dev/gfdl/ocean_only/single_column) convection experiments using MOM6-EPBL.

Notice that single column ocean only experiments are forced with the following variables:
- Net long wave
- Net shortwave
- Precipitation
- Evaporation
- Sensible heat 
- $\tau_x$
- $\tau_y$
-  snow
- runoff
- calving

So the heat-driven convection case will be forced with negative Sensible heat fluxes, and the salt-driven convection will be forced with evaporation fluxes:
<img width="719" alt="Screenshot 2024-02-09 at 3 13 00 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/99809abb-5617-40e7-b053-981ebc1c8d32">

**Comment by willaguiar** [comment]
**Case 1 - Salt-driven convection (Southern Ocean analogue)**

_________
Setup

Here I used the Weddell Sea surface fluxes from *21mbath* (ctrl) experiment to force the model, because in the Weddell Sector the SWMT response to thickening the top cell is [very evident](https://github.com/willaguiar/DSW-collaborative-project/issues/20#issuecomment-1769579013). Ocean initial conditions were set up as mean summer/Jan conditions in the Weddell Shelf (pink/black profiles below). Maximum depth for the experiment  is 500m. Based on total winter freshwater fluxes ([c] taken from SWMT code, including ice fluxes) we set up six experiments with evaporation fluxes ranging from $7 \times 10^{-5}\ kg\ m^{-3}\ s^{-1}$ until $1.5 \times 10^{-3}\ kg\ m^{-3}\ s^{-1}$, which are all values that occur in OM2 in the Weddell Sea[c]. We also identified the heat fluxes for the locations where the freshwater fluxes match our experiments, so we can use to force the model too ( Mostly for consistency, since heat fluxes are very small). To make sure that the ocean-only online calculated heat fluxes match the ones in OM2, we then adjusted the latent heat flux of evaporation. Adjusting the latent heat seemed to work, as the output net heat fluxes roughly matched the target heat fluxes (i.e., see [second fig in this code](https://github.com/willaguiar/DSW-collaborative-project/blob/main/JupyterCodes/EDIT_singlecolumn_forcing.ipynb))
![output](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/4e2d43e4-fc4f-426f-a99b-7e8c76a807ad)
<img width="691" alt="Screenshot 2024-02-09 at 3 40 06 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/ae376fa1-a6ee-4f0d-9c3a-eee883d444f2">

_________
Results

From top to bottom below is [a] MLD in the control, [b] $\Delta MLD$ as the difference from 5mtop -  1mtop, [c] $\Delta \sigma_0$ at surface as the difference from 5mtop -  1mtop, and [d] the same as [c] but zoomed. Notice that MLD seem to have only small peaking responses to thickening the top cell [b]. Regarding the surface $\Delta \sigma_0$, it gets progressively negative as we increase the FWF, i.e., it seems that thinner top cells indeed have bigger surface $\sigma_0$ in this convection setting, with the difference in surface $\sigma_0$ being proportional to the freshwater forcing.
![Weddell_exp](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/e2ae2944-e32f-4b61-890a-5e725a6bb65b)
Below is a plot of the FWF on $x$ and $\Delta \sigma_0$ on $y$, just to show that the response of surface $\sigma_0$ to thickening the top cell is proportional to the freshwater flux.
![Wconvection_reg](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/8596141f-f5e9-463a-a3cc-684144cb704c)
The problem however is that even tho the direction of density change here agrees with our OM2 Weddell findings, the magnitude of the signal is small, i.e, maximum $\sigma_0$ of DSW formation increases by ~ $0.1\ kg\ m^{-3}$ when cells are reduced from [5m to 1m](https://github.com/willaguiar/DSW-collaborative-project/blob/main/figs/SWMTsimulations_comp_by_sector_sigma0_total.png?raw=true), while here in the single column model the maximum response we find is of $0.016\ kg\ m^{-3}$


_Any ideas on what could be causing this difference? Also is there a better experimental set up we can do here?_

**Comment by adele-morrison** [comment]
I think this is ok. In the realistic model, the surface forcing is evolving in time due to the ocean / sea ice feedbacks. i.e. As the model produces more dense water (in the 1m case), it brings more CDW heat onto the shelf, which leads to a larger cooling flux and more sea ice formation / brine rejection. This causes the 1m and 5m cases to diverge, because they are experiencing different surface forcing. 

However in the single column case, the forcing is by definition fixed to be the same for both the 1m and 5m cases, because we are missing the feedbacks. So there is a limit to how large the density difference can get in the single column case.

I would be happy just putting the above justification in a paper for the low magnitude response of the single column experiments. What do others think? Do we need an idealised case that includes these feedbacks and has a magnitude of response that matches the realistic case? @AndyHoggANU @dkhutch ?

**Comment by AndyHoggANU** [comment]
It's a good point, @adele-morrison , that we are missing the feedback from melting away ice. I agree that it's reasonable to argue that this small difference could easily trigger larger differences. If you didn't trust that, I think the next step would be to code a single column model with feedback and sea ice ... which is a lot harder!

**Comment by willaguiar** [comment]
I was making some tests to see if I could set up these experiments in a way that makes more sense and made some changes.

**Comment by willaguiar** [comment]
I made some changes to see if I could set up these experiments in a way that makes more sense .

The changes are:
-Surface  forcing:  In the new single column experiments I added the evaporation fluxes only in the South 2 cells only ( north 2 cells have no fluxes). I additionally made the domain non-reentrant. The idea is to create convection with downwards movement in the south, upwards in the north, and lateral advection at surface. 
-IC: The south cells have now inner Weddell shelf IC conditions (black/pink lines in the profile of the first comment), and the North cells have outer shelf IC (green/purple lines). There are warmer waters on the surface at the North cells.

This would not simulate the sea ice feedback, but at least we can have lateral advection of warmer waters  for the beginning of the simulation giving more amplitude for the density change.... Comparing experiments with the same evaporation ($1.5 \times 10^{-3}\ Kg\ m^{-2}\ s$)  but with the changes said above (dotted brown lines) and without it (full brown line), we see that these new changes have a small increase in density anomalies.

Finally, I was thinking that I used the monthly mean OM2 freshwater/salt flux outputs to define the evaporation fluxes that I would apply in the single column model. But I imagine some days might have much stronger sea ice formation (stronger winds, colder atm) than others and the monthly mean forcing might just be a lowball. I didn't have daily freshwater fluxes saved in our experiments, but I looked at July 2017 in iaf cycle 1 ( which have these daily fluxes), and there are days that the FWF on the coast are double the monthly mean fluxes. So I did a new experiment with this new higher FWforcing  limit found in the Weddell Sea (i.e., $3 \times 10^{-3}\ Kg\ m^{-2}\ s$, gold line and gold dot), and found a higher value for the density change($-0.03\ kg\ m^{-3}$). 

Just letting you know cause I think this result shortens the difference between the SWMT signal and surface density signal we saw before. 
making it easier to justify that the remaining difference could be due to sea ice feedback.


<img width="613" alt="Screenshot 2024-02-15 at 5 40 02 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/086f86d9-24c8-4198-8c97-f75ea94b8351">
<img width="514" alt="Screenshot 2024-02-15 at 5 40 27 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/2f882f3b-ef03-4e65-a5dc-79512270acd9">

PS: I did a manuscript outline [here](https://github.com/willaguiar/DSW-collaborative-project/blob/main/manuscript_outline.md) for this study, so we can start thinking on what images and results to include for the submission

**Comment by adele-morrison** [comment]
Yes, I think these higher fluxes combined with the fact we have no feedback is very justifiable to explain the difference. If you wanted to see how much the max fluxes vary year to year there is daily output for most of an IAF cycle in 01deg_jra55v150_iaf_cycle1.

Not sure about explaining the North Atlantic though?
